lib/circles/create.js

```diff
@@ -1,23 +1,23 @@
 const fullStateEnum = require("./fullStateEnum");
 
 module.exports = function(req, res) {
-    if(!req.body.name) return res.status(401).send({ status: "Circle name is missing" });
-
     const circle = {
         name: req.body.name || null,
         vision: req.body.vision || null,
         mission: req.body.mission || null,
         aim: req.body.aim || null,
         expectationsForMembers: [],
-        members: [{ id: req.user.id, invitationState: "member" }],
+        members: [req.user.id],
+        invited: req.body.invited || [],
         contactPerson: req.user.id,
         fullState: req.body.fullState || "lookingForMore",
     };
 
-    if(circle.fullState && !fullStateEnum.includes(circle.fullState)) {
-        req.log.warn("Tried to update circle with invalid fullstate");
-        return res.status(400).send({ status: `fullState can only be the following values: ${JSON.stringify(fullStateEnum)}` });
-    }
+    if(!circle.name) return res.status(400).send({ status: "Circle name is missing" });
+    if(!(circle.invited instanceof Array)) return res.status(400).send({ status: "invited must be a list of userIds" });
+    if(circle.fullState && !fullStateEnum.includes(circle.fullState)) return res.status(400).send({ status: `fullState can only be the following values: ${JSON.stringify(fullStateEnum)}` });
```

Comment:
Some very long lines! Ok for now, but maybe think of how they can be made easier to read :smile: